.gitlab-ci.yml

```diff
@@ -371,8 +371,9 @@ build-push-windows-image:
         $IMAGE_TAG = $env:CI_COMMIT_SHA
       }
       $SMART_AGENT_RELEASE = $((Get-Content internal\buildscripts\packaging\smart-agent-release.txt).TrimStart("v"))
+      $JMX_LIB_VERSION = $(Get-Content internal\buildscripts\packaging\jmx-lib-version.txt)
```

Comment:
since the agent version introduced a `_RELEASE` pattern I'd suggest we use that over `_VERSION`

Reply:
Changes made.
Thanks for the correction!